README.md

```diff
@@ -176,6 +176,7 @@ key is `urls`. Other optional options are:
   of strings for headless Chrome](https://peter.sh/experiments/chromium-command-line-switches/).
 * `cssoOptions` - CSSO compress function [options](https://github.com/css/csso#compressast-options)
 * `timeout` - Maximum navigation time in milliseconds, defaults to 30 seconds, pass 0 to disable timeout.
+* `ignoreCssError` - ignore CSS errors. By default minimalcss reports all CSS errors, but if you want to ignore those errors, if you do not have access to those css files, you can set this option to true.
```

Comment:
Sorry for being difficult but that should really be called `ignoreCssErrors` (with a `s` suffix) because the functionality is that it will ignore all CSS errors for the whole duration of the `minimalcss` run.

Reply:
Also, can you uppercase the second instance of the word `css` to `CSS`. For consistency.

Reply:
Yes sure. I was exactly expecting help on naming